vue-components/src/components/ValidationMessage.vue

```diff
@@ -50,7 +50,7 @@ export default Vue.extend( {
 	line-height: $wikit-ValidationMessage-line-height;
 	padding-block-start: $wikit-ValidationMessage-padding-block-start;
 	display: inline-flex;
-	align-items: flex-start;
```

Comment:
I believe this shouldn't be changed. It makes the icon to center-align with the validation message when the latter wraps, while it should preserve its alignment at the top.

<img width="673" alt="Screenshot 2021-01-19 at 11 42 25" src="https://user-images.githubusercontent.com/3887458/105023564-68122a00-5a4b-11eb-9509-f3345d376793.png">

Reply:
Do you want me to look into this? Also, I think it would make sense to have one example with such a long error message in a story as these are exactly the things that the visual regressions tests are supposed to catch.

Reply:
I think it's a matter of undoing this change, since only the value is changing, right? I'm not sure of the conceptual implications, but it sounds better to me that the alignment of the children is defined in the parent's code (rather than inline, at the story level).

About the copy, you're absolutely right, and I'll update it.

Reply:
Yes, this change and also re-add the padding below.

Reply:
Yes, this change and also re-add the padding below.